lib/griddler/sendgrid/adapter.rb

```diff
@@ -61,13 +61,28 @@ def bcc_from_envelope(envelope)
       end
 
       def attachment_files
-        params.delete('attachment-info')
-        attachment_count = params[:attachments].to_i
+        params[:attachments].to_i.times.map do |index|
```

Comment:
What are your thoughts on keeping the `params[:attachments].to_i` as `attachment_count` it's not as clear to me what's happening here.